templates/patient/card/card_base.html.twig

```diff
@@ -1,7 +1,7 @@
 <section class="{{ card_container_class_list|default(['card', 'mb-2'])|join(' ') }} {{ card_bg_color }} {{ card_text_color }}">
```

Comment:
this must have the card class for styles to work and as container for card body.
I have all this in my PR

Reply:
nice! Sounds like I can drop this PR then?

Reply:
yep, The whole OemrUI logic got factored out when the user setting save post was changed, so for demographic that function will fail on save. The last refactor needed is container (demographics_ps_expand)
working it now